PyDSS/value_storage.py

```diff
@@ -125,91 +165,69 @@ def __init__(self, name, prop, values, label_suffixes):
             Pairs of values that can be interpreted as complex numbers.
 
         """
+        super().__init__()
         self._name = name
         self._prop = prop
         self._labels = []
         self._data = {}
+        self._value_type = None
+        self._value = []
 
         assert (isinstance(values, list) and len(values) == len(label_suffixes)), \
             '"values" and "label_suffixes" should be lists of equal lengths'
         for val, lab_suf in zip(values , label_suffixes):
             label = prop + '__' + lab_suf
             self._data[label] = [val]
             self._labels.append(label)
+            self._value.append(val)
```

Comment:
@AadilLatif Can you confirm that this is correct?

Reply:
Yes you are correct